pyscriptjs/src/pyscript.py

```diff
@@ -104,6 +104,22 @@ def format_mime(obj):
     return MIME_RENDERERS[mime_type](output, meta), mime_type
 
 
+def display(*objects, parent=None):
+    """Display `objects` on parent.
+    All non-keyword arguments are converted using the global renderer mapper or using the
+    object `__repr__` if a renderer for that object is not present.
+    If `parent` is None, the parent object set at the global `py-script` tag level will be used.
+    """
+    parentElement = None
+    append = False
+    if parent is None:
+        parentElement = document.getElementsByTagName("py-script")[0].parentElement
```

Comment:
I'm not a JS expert and I might be missing something, but this looks wrong?
If we have multiple `<py-script>` tags, this would always overwrite the content of the first one, wouldn't it?

Reply:
Should i implement the behaviour you discussed in the task or rather wait for more opinions?

Reply:
> Should i implement the behaviour you discussed in the task or rather wait for more opinions?

I think it's a good idea to wait for more opinions until we settle down and decide what API we really want. @fpliger ?